python/tests/domains/test_qnn.py

```diff
@@ -0,0 +1,48 @@
+# ============================================================================ #
+# Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                   #
+# All rights reserved.                                                         #
+#                                                                              #
+# This source code and the accompanying materials are made available under     #
+# the terms of the Apache License 2.0 which accompanies this distribution.     #
+# ============================================================================ #
+import os
+
+import pytest
+import numpy as np
+
+import cudaq
+
+
+def test_simpleObserveN_QNN():
+    n_qubits = 2
+    n_samples = 5000
+    h = cudaq.spin.z(0)
+    n_parameters = n_qubits * 3
+    parameters = np.random.default_rng(13).uniform(low=0,
+                                                   high=1,
+                                                   size=(n_samples,
+                                                         n_parameters))
+    np.random.seed(1)
+
+    kernel, params = cudaq.make_kernel(list)
```

Comment:
Would you mind adding a comment or two about this kernel? I'm not too familiar with the QNN circuits